app/google_api_parser.py

```diff
@@ -0,0 +1,26 @@
+import json
+import requests
+from config import GOOGLE_API_KEY
+
+
+geocode_url = 'https://maps.googleapis.com/maps/api/geocode/json?'
+
+
+class GoogleGeocode:
+
+    def __init__(self):
+        self.formatted_address = None
+        self.place_type = None
+
+    def get_geocode_request(self, address):
+        params = {'address': address, 'key': GOOGLE_API_KEY}
+        return json.loads(requests.get('{}'.format(geocode_url), params=params).text)
+
+    def get_place_info(self, address):
+        geocode = self.get_geocode_request(address)
+        status = geocode['status']
+        if status == 'OK':
+            self.place_type = geocode['results'][0]['address_components'][0]['types'][0]
+            self.formatted_address = geocode['results'][0]['formatted_address']
+            return {'place_type': self.place_type, 'formatted_address': self.formatted_address}
+        return status
```

Comment:
but instead, i'd just return None. This weird status-string is specific to this API. you should try to avoid leaking API-details outside this module.